### PR TITLE
Update Set-ADUser.md

### PR DESCRIPTION
**Pull request by Nephiaust**
Added note to -Country to specify the use of the ISO-3166 2 character country code & linked to AD Schema definition.

### CODE REVIEW

File: docset/winserver2019-ps/activedirectory/Set-ADUser.md [hunk 1]
```diff
@@ -496,6 +496,7 @@ Accept wildcard characters: False
 Specifies the country or region code for the user's language of choice.
 This parameter sets the **Country** property of a user object.
 The LDAP display name (**ldapDisplayName**) of this property is c.
+This value uses the ISO-3166 2 character code (see https://docs.microsoft.com/en-us/windows/win32/adschema/a-c).
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/activedirectory/Set-ADUser.md:499]:
```suggestion
This value uses the ISO-3166 2-character code (see [Country-Name attribute](/windows/win32/adschema/a-c).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Nephiaust, kindly commit the suggested changes. Many thanks.

**Comment by Nephiaust** [comment]
> Copy review - @Nephiaust, kindly commit the suggested changes. Many thanks.

Cheers :)

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]
Can you please check and make the same edit to the other OS Versions of the cmdlet article? Thanks

**Review by tseward — APPROVED** [review]

File: docset/winserver2019-ps/activedirectory/Set-ADUser.md [hunk 2]
```diff
@@ -496,6 +496,7 @@ Accept wildcard characters: False
 Specifies the country or region code for the user's language of choice.
 This parameter sets the **Country** property of a user object.
 The LDAP display name (**ldapDisplayName**) of this property is c.
+This value uses the ISO-3166 2-character code (see [Country-Name attribute](/windows/win32/adschema/a-c).
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/activedirectory/Set-ADUser.md:499]:
```suggestion
This value uses the ISO-3166 2-character code (see [Country-Name attribute](/windows/win32/adschema/a-c)).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @Nephiaust, one more minor change, please.

**Review by dariomws — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**